docs/Demos.md

```diff
@@ -7,6 +7,196 @@ title: "React-admin demos"
 
 If you want to see what react-admin is capable of, or if you want to learn from apps built by seasoned react-admin developers, check out these demos.
 
+## Overview
+
+Discover our demos that show you the react-admin potential and help you familiarize with it.
+
+<style>
+    .demos-list {
+        display: grid;
+        grid-template-columns: repeat(2, 1fr);
+        grid-template-rows: repeat(3, 1fr);
```

Comment:
I'm not a css grid specialist, but I'm a bit surprised you have to set both `grid-template-columns` and `grid-template-rows`. To me, you could keep only `grid-template-columns`.